### PR TITLE
MUMUP-2407: Priority Notification

### PR DESCRIPTION
**Pull request by jhanstra**
Goes along with https://github.com/UW-Madison-DoIT/uw-frame/pull/173

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/my-app/layout/list/partials/home-list-view.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <div class="row portlet-frame">
+  <notification-bell mode="priority" class="visible-xs"></notification-bell>
```
Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/layout/list/partials/home-list-view.html]: Is there any way we can make this not be in the header of every single page? Maybe in the app-header or in the frame page?

**Comment by apetro** [comment]
Observation: seems like `<notification-bell mode="priority" class="visible-xs"></notification-bell>` is being sprinkled into several partials. Opportunity to refactor to use fewer partials to implement the top part of the page, the neighborhood of the priority notification display?

**Comment by jhanstra** [comment]
@apetro @timlevett it doesn't look like those partials were even necessary, since priority notifications are already attached to the header. I removed them. Now this changeset is really just supplying a few notifications as priority.

**Comment by apetro** [comment]
:+1:

**Comment by timlevett** [comment]
:+1:

**Pull request merged**